src/main/scala/cromwell/Main.scala

```diff
@@ -20,16 +21,40 @@ object Actions extends Enumeration {
 }
 
 object Main extends App {
-  val Properties = System.getProperties
-  val LoggerProperty = "CROMWELL_LOGGER"
-  lazy val Log = LoggerFactory.getLogger("main")
 
-  Option(Properties.getProperty(LoggerProperty)) match {
-    case None if args.headOption.map {_.toUpperCase}.contains("SERVER") =>
-      Properties.setProperty(LoggerProperty, "SERVER")
+  val LogRootDefault = "."
```

Comment:
all of this stuff feels like it shouldn't live directly in the object's constructor